### PR TITLE
feat: cockerilltim changes to best practices

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Restore proposed Best Practices changes from @cockerilltim.

## Changes

- **added** paragraph and link under "Best Practices"

## Notes

> [!NOTE]
> On 2024-04-25, there was a mishap on the DS-User-Guide repository. These changes were in a PR that was lost.

### CODE REVIEW

**Review by cockerilltim — APPROVED** [review]

File: user-guide/docs/curating/guides.md [hunk 1]
```diff
@@ -133,6 +133,8 @@ Finally, click <strong>Request DOI &amp; Publish</strong> and agree to the agree
 
 ### [Simulation](#simulation) { #simulation }
 
+We recommend reading the Best Practices Guide for [Simulation Data](../../bestpractices/working#simulationdata) as a prerequesite to initiating your project.
```
Comment (wesleyboar) [inline user-guide/docs/curating/guides.md]: Consider wording that is less verbose. User might understand faster. Also, the link was not working, I found this one to work.

```suggestion
Read the [Simulation Data Best Practices Guide](../../curating#bestpractices-simulationdata) prior to initiating your project.
```

File: user-guide/docs/curating/guides.md [hunk 2]
```diff
@@ -133,6 +133,8 @@ Finally, click <strong>Request DOI &amp; Publish</strong> and agree to the agree
...
+
 #### [1. Add a Project](#simulation-step1) { #simulation-step1 }
 
 You can start a project at the very beginning of its lifespan, upload and curate data incrementally, then publish sets of data at your convenience.
```
Comment (wesleyboar) [inline user-guide/docs/curating/guides.md:140]: Test.

**Review by wesleyboar — COMMENTED** [review]
Good addition. One note. Ignore the test.

File: user-guide/docs/curating/guides.md (hunk 1 above)
Comment (susanunit) [reply user-guide/docs/curating/guides.md]: excellent

**Review by susanunit — APPROVED** [review]

**Pull request merged**